feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go

```diff
@@ -294,40 +292,47 @@ const (
 	dynamic        = telemetry.IfIp_NeighborOrigin_DYNAMIC
 )
 
-func (tc *testCase) verifyLagID(t *testing.T, dp *ondatra.Port) {
+func (tc *testCase) verifyAggID(t *testing.T, dp *ondatra.Port) {
 	dip := tc.dut.Telemetry().Interface(dp.Name())
 	di := dip.Get(t)
 	if lagID := di.GetEthernet().GetAggregateId(); lagID != tc.aggID {
 		t.Errorf("%s LagID got %v, want %v", dp, lagID, tc.aggID)
 	}
 }
+
 func (tc *testCase) verifyInterfaceDUT(t *testing.T, dp *ondatra.Port) {
 	dip := tc.dut.Telemetry().Interface(dp.Name())
 	di := dip.Get(t)
 	fptest.LogYgot(t, dp.String(), dip, di)
+
+	// LAG members may fall behind, so wait for them to be up.
+	dip.OperStatus().Await(t, time.Minute, opUp)
```

Comment:
di needs to be collected after the Await (or just use the values returned by Await) doesn't it?  If the Await is doing anything, the values saved in di are the old states, and they will just fail in the next few lines even if the interface does come up after a delay.

Reply:
But if I move the dip.Get(t) after the await, then if the interface is down, then await will fatal out and di is never logged.

Reply:
Removed the di.GetOperStatus() check which used potentially stale value.